src/services/user/hooks/index.js

```diff
@@ -168,7 +170,7 @@ const checkJwt = () => function checkJwtfnc(hook) {
 
 const pinIsVerified = (hook) => {
 	if ((hook.params || {}).account && hook.params.account.userId) {
-		return (globalHooks.hasPermission('USER_CREATE')).call(this, hook);
+		return (globalHooks.hasPermission(['STUDENT_CREATE', 'TEACHER_CREATE', 'ADMIN_CREATE'])).call(this, hook);
```

Comment:
why wee need call and can not execute globalHooks.hasPermission(['STUDENT_CREATE', 'TEACHER_CREATE', 'ADMIN_CREATE']))(context) ?